libs/src/services/solanaWeb3Manager/transactionHandler.js

```diff
@@ -124,24 +154,70 @@ class TransactionHandler {
       })
     }
 
+    const rawTransaction = tx.serialize()
+
+    // Send the txn
+
+    const sendRawTransaction = async () => {
```

Comment:
instead of just using connection.sendRawTransaction a la https://github.com/AudiusProject/audius-protocol/blob/b06486d24349a4764fbc8296e1564f3f557d9321/identity-service/src/solana-client.js#L253

Reply:
defined it here since we use it in two places (initial send + polling loop), and it's a ton of fiddly params and stuff to pass through each time. easier to just capture it in a single function to call